flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/serializer/JsonDebeziumSchemaSerializer.java

```diff
@@ -105,6 +109,7 @@ public JsonDebeziumSchemaSerializer(DorisOptions dorisOptions,
         this.firstLoad = true;
         this.firstSchemaChange = true;
         this.schemaChangeManager = new SchemaChangeManager(dorisOptions);
+        this.dorisSystem = new DorisSystem(dorisOptions);
```

Comment:
DorisSystem needs to pass in jdbc-url, which was not required in previous versions. Can it be made compatible?